ci-operator/config/openshift/ansible-operator-plugins/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
+  - joelanford
+  - everettraven
+  - oceanc80
+  - TrilokGeer
+reviewers:
+  - joelanford
+  - everettraven
+  - oceanc80
+  - TrilokGeer
```

Comment:
You can add both your and my github usernames in the reviewers list.

Reply:
Done, added